.github/in-cluster-test-scripts/external-workloads-install.sh

```diff
@@ -11,6 +11,9 @@ cilium install \
   --kube-proxy-replacement=strict \
   --native-routing-cidr="${CLUSTER_CIDR}"
 
+# Enable Relay
+cilium hubble enable
```

Comment:
Shouldn't we move this below `cilium status --wait`? We had an issue caused by Cilium pods not coming up fast enough when pods created by `cilium clustermesh enable` were not managed by Cilium.

I am not sure whether Relay pod needs to be managed by Cilium, but I assume that that's the case.

Reply:
We seem to do it in this order (i.e. `cilium hubble enable`, then `cilium status --wait`) in most other test jobs. Presumably because we want to wait for Hubble relay to become available and managed by Cilium. I'm not sure I understand the issue with `cilium clustermesh enable`. I see that `cilium status --wait` appears before we enable ClusterMesh, so we should be safe in that regard. Or am I missing something?

Reply:
Regarding having `cilium status --wait` really wait for Hubble relay to be available also see #365 which should help in cases where `cilium status --wait` would return even though Hubble relay wasn't fully available, i.e. still reporting warnings.

Reply:
I think we might need to have two `cilium status --wait`. As @nebril pointed out correctly, it is important that `clustermesh-apiserver` and `hubble-relay` are managed by Cilium, meaning the CNI needs to be installed properly before the deployment is started. So I think it does make sense to first have a `cilium status --wait` just for Cilium, then do a `cilium hubble/clustermesh enable`, and then do another `cilium status --wait` (with #365 applied) to ensure the restarted cilium pods, clustermesh-apiserver and hubble-relay are all ready to go.

Reply:
Thanks, I'ill add a second `cilium status --wait` before `cilium hubble enable` in that case. I think the `clustermesh-apiserver` readiness check will be taken care of by `cilium clustermesh status --wait` further down, also see commit c171080ef434.

Reply:
fyi `cilium hubble enable` restarts cilium and wait for it to become ready before deploying hubble relay / ui: https://github.com/cilium/cilium-cli/pull/335 so the first `cilium status --wait` is probably not strictly required.

Reply:
Thanks. I've checked that `cilium hubble enable` will indeed wait for Cilium to become fully ready before proceeding:

https://github.com/cilium/cilium-cli/blob/619f9d5a7b799e3a43ec6af42ac7a3f3e1d83c8d/hubble/hubble.go#L277-L282

So the first `cilium status --wait` is indeed not necessary. I've dropped it again.